ruler-gradle-plugin/src/main/kotlin/com/spotify/ruler/plugin/apk/ApkSanitizer.kt

```diff
@@ -158,7 +167,16 @@ class ApkSanitizer(
                 entry.name.startsWith("/lib/") -> FileType.NATIVE_LIB
                 else -> FileType.OTHER
             }
-            return AppFile(entry.name, type, entry.downloadSize, entry.installSize)
+            val resourceType: ResourceType? = when {
```

Comment:
I think it's worth it to add some tests to check if this works correctly (i.e. one for `DRAWABLE` and one for `OTHER`).

Reply:
Yeah, Added some new and updated the existing ones!